src/components/inputBar/inputBar.component.tsx

```diff
@@ -0,0 +1,17 @@
+type InputBarProps = {
+	name: string;
+	type: string;
+	placeHolder: string;
+};
+
+function InputBar(props: InputBarProps) {
```

Comment:
Props are not de structured inside the function. They are de structured in the function parameters. 
```suggestion
const InputBar = ({name, type, placeHolder}: InputBarProps) => {
```